src/Azure.Functions.Cli/StaticResources/package-ts-v4.json

```diff
@@ -6,16 +6,18 @@
   "scripts": {
     "build": "tsc",
     "watch": "tsc -w",
-    "prestart": "npm run build",
+    "clean": "rimraf dist",
+    "prestart": "npm run clean && npm run build",
```

Comment:
Doesn't running `npm run clean && npm run build` still run into the same issue of not being compatible on all shells (particularly PowerShell)? I thought that was the main argument against putting `rimraf dist &&` directly in the `prestart` script

Reply:
Yeah that was a concern, but turns out PowerShell added support for those in PowerShell 7 (first released in 2020 I think)

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_pipeline_chain_operators